Source/Resources/Content/AzureManagement/MicrosoftResources/ResourceGroups/m365logging.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
+  "description":"What is it: Creates an Azure Event Hub, Azure Log Analytics Workspace, and Azure Storage account to capture Azure and Intune logs. \n Simeon Notes: These are the essential resources required to capture logs, analyze trends, and be alerted of suspicious events. Simeon's baseline Workspace is named 'tenant Name-M365logging' \n Learn More: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/design-logs-deployment",
```

Comment:
**What does this do?**
Creates an Azure Event Hub, Azure Log Analytics Workspace, and Azure Storage account to capture Azure and Intune logs. The baseline Workspace is named 'tenant Name-M365logging'. 

**Why should you use this?**
This provides the essential resources required to capture logs, analyze trends, and be alerted of suspicious events.

**What is the end-user impact?**
N/A

**Learn more**
[Designing your Azure Monitor Logs deployment](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/design-logs-deployment)